app/assets/javascripts/angular/controller/forms.js.erb

```diff
@@ -1,3 +1,26 @@
-app.controller('ListFormsCtrl', ['$scope', function($scope){
+app.controller('ListFormsCtrl', ['$scope', '$http','$auth','$location', function($scope, $http, $auth, $location){
   
+}]);
+
+app.controller('FormsCreateCtrl', ['$scope', '$http','$auth','$location', function($scope, $http, $auth, $location){
+  
+  $scope.add = function(elem) {
+    var html = '';
+    html += '<li><a href="/#/forms/create">'
+    html +=  elem
+    html += '</a></li>'
+    $('#formStuff').append(html);
+  };
+
+}]);
+
+app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$routeParams', function($scope, $http, $auth, $location, $routeParams){
+  
+  url = "<%= ENV['URL'] %>" || 'http://localhost:3000'
+
+  $http.get(url + '/forms/' + $routeParams.id).success(function(response) {
+    $scope.name = response.name;
```

Comment:
Maybe it's better to do `$scope.form = response`

Then you can do

`$scope.form.name` and `$scope.form.content`